apps/system/test/unit/mock_app.js

```diff
@@ -70,7 +70,13 @@ MockApp.prototype.mTriggerDownloadError = function(error) {
     });
   }
 };
+MockApp.prototype.addEventListener = function(evt, callback) {
+callback();
```

Comment:
Hi Marshall,

I tried doing this, but the event is attached to the app, vs. the window.  window.dispatchEvent seems to work nicely for all the events in AUM.  However, now, that the event is dispatched on the app, and the app uses this MockApp, I had to add add/removeEventListener and dispatchEvent to the MockApp, so they don't really do much right now.  Maybe there is something I'm missing here.